helm/README.md

```diff
@@ -0,0 +1,22 @@
+# Conjur Kubernetes Authentication Helm Charts
+
+## Overview
```

Comment:
Do we have any pictures showing how this works from the solution doc that would be helpful to add here?  Not required, just a thought.

Reply:
I couldn't find a relevant image :/

Reply:
Thats fine!  No worries.